libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -37,12 +37,15 @@
         View.OnTouchListener,
         EditorMediaUploadListener,
         IHistoryListener {
+    private static final String KEY_HTML_MODE_ENABLED = "KEY_HTML_MODE_ENABLED";
+
```

Comment:
nitpick: we may want to remove this extra line here and have all static final strings together

Reply:
Those constants have a very different meaning/function so I kept them separate. That said, no biggie bringing them closer. Done with 309d1504d9.